### ISSUE TITLE
Rules in configuration comments in vue files have no effect

### ISSUE THREAD

**Issue by ThomasR** [vuejs/eslint-plugin-vue#588]
**Tell us about your environment**

* **ESLint Version:** 5.5.0
* **eslint-plugin-vue Version:** 5.0.0-beta3
* **Node Version:** 10.10

**Please show your full configuration:**

```
{
    plugins: ['vue']
}
```

**What did you do? Please include the actual source code causing the issue.**

The docs suggest that you could use `<!-- eslint vue/…` or even `/*eslint vue/…` to set certain rules on-the-fly from within vue files. For example, these are taken from the docs:

```vue
/* eslint vue/singleline-html-element-content-newline: ["error", { "ignoreWhenNoAttributes": false}] */

<div>content</div>

<tr><td>{{ data1 }}</td><td>{{ data2 }}</td></tr>

<div><!-- comment --></div>
```

```vue
<!-- eslint vue/html-closing-bracket-newline: "error" -->

<div id="foo" class="bar">
<div
    id="foo"
    class="bar"
>
```

**What did you expect to happen?**

Inline rules should have an effect.

As a temporary solution, the docs should make absolutely clear that this does not work (yet).

**What actually happened? Please include the actual, raw output from ESLint.**

Nothing.

This is particularly confusing since `<!-- eslint-disable` and the like [work just fine](https://github.com/vuejs/eslint-plugin-vue/blob/master/docs/rules/comment-directive.md).

**Comment by michalsnik** [comment]
You're absolutely right @ThomasR This doesn't work this way, and it was only a way of showing rule configuration. We'll try to address this matter, as it in fact might be a bit confusing. Thanks for pointing it :)

**Comment by michalsnik** [comment]
We're in the process of creating new documentation. And it will be addressed here: #534

**Comment by armano2** [comment]
@ThomasR you can read more about this in https://vuejs.github.io/eslint-plugin-vue/rules/comment-directive.html
